tests/unit/via/views/route_by_content_test.py

```diff
@@ -28,41 +17,37 @@ class TestRouteByContent:
         ),
     )
     def test_it_routes_by_content_type(
-        self, content_type, location, call_route_by_content
+        self, content_type, location, call_route_by_content, get_url_details
     ):
-        result = call_route_by_content(content_type)
+        get_url_details.return_value = (content_type, 200)
+        result = call_route_by_content()
```

Comment:
Previously we had a more complicated `call_route_by_content` which setup an HTTP pretty call etc. Now we can just mock `get_url_details` as an external dependency.